src/index.js

```diff
@@ -9,11 +10,13 @@ export default function () {
         currentFixtureName: null,
         testCount:          0,
         singleVideoHash:    null,
+        options:            null,
 
-        async reportTaskStart (startTime, userAgents, testCount) {
+        async reportTaskStart (startTime, userAgents, testCount, testStructure, { configuration } = {}) {
             this.startTime = startTime;
             this.uaList    = userAgents.join(', ');
             this.testCount = testCount;
+            this.options   = configuration;
```

Comment:
I feel that it's better to store only the necessary information here, e.g. `screenshotPath` & `videoPath`. If we do it this way, we will need to check `configuration` existence only a single time.